docs/content/text/textProperties.rst

```diff
@@ -12,6 +12,7 @@ Text Properties
 .. autofunction:: drawBot.listOpenTypeFeatures
 .. autofunction:: drawBot.fontVariations(wdth=0.6, wght=0.1, ...)
 .. autofunction:: drawBot.listFontVariations
+.. autofunction:: drawBot.listFontVariationNamedInstances
```

Comment:
I wonder if `listNamedInstances()` would be good enough?

Reply:
also fine for me: named instances only appear within var fonts (that the only reason why I added the long method name